server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -224,7 +224,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.indexing.docs.current",
+                "es.node.stats.indices.indexing.docs.total",
```

Comment:
Are we sure this is total and not count? I see the description names "Current". Maybe is it the description that needs update instead?

Reply:
good catch. I think this has to keep the current subgroup to indicate this is about 'in-flight indexing operations'